R/chat_bot.R

```diff
@@ -0,0 +1,22 @@
+chat_bot <- function(system_prompt = NULL, default_turns = list()) {
+  system_prompt <- system_prompt %||% databot_prompt()
```

Comment:
Is `%||%` from rlang? I'm getting

```
Error in `system_prompt %||% databot_prompt()`:
! could not find function "%||%"
```

Reply:
It's also in R itself. What version of R are you using?

Reply:
4.3.2. I didn't realize how old that is now